IDEAS/Buildings/Components/Shading/HorizontalFins.mo

```diff
@@ -19,32 +22,50 @@ model HorizontalFins "horizontal fins shading"
 
 
 protected
-  Modelica.SIunits.Length dy1 = s-cos(beta_internal)*w-sin(beta_internal)*t;
+  Modelica.SIunits.Length dy1 = s-sin(beta_internal)*w-cos(beta_internal)*t;
```

Comment:
@icupeiro I think I spotted a bug here. Can you verify that you agree with this change?

Reply:
agreed